src/rebuild/rebuild_iv.c

```diff
@@ -136,17 +136,19 @@ rebuild_iv_ent_update(struct ds_iv_entry *entry, struct ds_iv_key *key,
 		rebuild_global_status_update(rgt, src_iv);
 		if (rgt->rgt_status.rs_errno == 0) {
 			rgt->rgt_status.rs_errno = src_iv->riv_status;
-			if (src_iv->riv_status != 0)
+			if (src_iv->riv_status != 0) {
 				rgt->rgt_status.rs_fail_rank = src_iv->riv_rank;
+				rgt->rgt_abort = 1;
+			}
 		}
-		D_DEBUG(DB_TRACE, "update rebuild "DF_UUID" ver %d gen %u"
-			"toberb_obj/rb_obj/rec/global state/status/rank "
-			DF_U64"/"DF_U64"/"DF_U64"/%d/%d/%d\n",
+		D_DEBUG(DB_REBUILD, "update rebuild "DF_UUID" ver %d gen %u"
+			"toberb_obj/rb_obj/rec/global state/status/rank/abort "
```

Comment:
(style)  break quoted strings at a space character

Reply:
Line 144 and 145 will place the %u with no space before the toberb_obj